src/simpledi.js

```diff
@@ -98,17 +98,38 @@ SimpleDi.withNew = function(Constructor) {
   };
 };
 
+SimpleDi._withoutNew = function(factory) {
```

Comment:
Right, I think there is no use for a `SimpleDi.withoutNew` factory helper, so I'll make it private.